extensions/api/data-management/asset/src/test/java/org/eclipse/dataspaceconnector/api/datamanagement/asset/model/AssetDtoTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 Diego Gomez
```

Comment:
As far as I know the Copyright should go to ZF, at least that is my latest information.